docs/CONTRIBUTING-id.md

```diff
@@ -22,31 +25,230 @@ Dengan partisipasi Anda, Anda berjanji untuk mengikuti [Kode Etik](CODE_OF_CONDU
+- Video YouTube. Kami biasanya tidak menautkan ke video YouTube individu kecuali jika durasinya lebih dari satu jam dan terstruktur seperti kursus atau tutorial.
+- Leanpub. Leanpub menyelenggarakan buku dengan berbagai model akses. Terkadang sebuah buku dapat dibaca tanpa registrasi; terkadang sebuah buku memerlukan akun Leanpub untuk akses gratis. Mengingat kualitas buku dan campuran dan fluiditas model akses Leanpub, kami mengizinkan daftar yang terakhir dengan catatan akses `*(Akun Leanpub atau email yang valid diminta)*`.
+
+
+<!----><a name="genres"></a>
+#### Genre
+
+Aturan pertama dalam memutuskan daftar mana yang termasuk dalam sumber daya adalah melihat bagaimana sumber daya itu menggambarkan dirinya sendiri. Jika itu menyebut dirinya sebuah buku, maka mungkin itu adalah sebuah buku.
+
+
+<!----><a name="genres-we-dont-list"></a>
+##### Genre yang tidak kami cantumkan
+
+Karena Internet sangat luas, kami tidak memasukkan dalam daftar kami:
+
+- blog
+- postingan blog
+- artikel
+- situs web (kecuali yang menghosting BANYAK item yang kami daftarkan).
+- video yang bukan kursus atau screencasts.
+- bab buku
+- sampel penggoda dari buku
+- Saluran IRC atau Telegram
+- Celana panjang atau milis
+
+Daftar pemrograman kompetitif kami tidak seketat pengecualian ini. Lingkup repo ditentukan oleh komunitas; jika Anda ingin menyarankan perubahan atau penambahan ruang lingkup, silakan gunakan masalah untuk membuat saran.
+
+
+<!----><a name="books-vs-other-stuff"></a>
+##### Buku vs. Barang Lainnya
+
+Kami tidak rewel tentang kebukuan. Berikut adalah beberapa atribut yang menandakan bahwa sumber daya adalah sebuah buku:
+
+- memiliki ISBN (Nomor Buku Standar Internasional)
+- memiliki Daftar Isi
+- versi yang diunduh, terutama ePub, ditawarkan
+- memiliki edisi
+- itu tidak tergantung pada konten atau video interaktif
+- mencoba untuk mencakup topik secara komprehensif
+- itu mandiri
+
+Ada banyak buku yang kami daftarkan yang tidak memiliki atribut ini; itu bisa tergantung pada konteksnya.
+
+
+<!----><a name="books-vs-courses"></a>
+##### Buku vs. Kursus
+
+Terkadang ini sulit untuk dibedakan!
+
+Kursus sering kali memiliki buku teks terkait, yang akan kami daftarkan dalam daftar buku kami. Kursus memiliki kuliah, latihan, tes, catatan atau alat bantu didaktik lainnya. Sebuah kuliah atau video dengan sendirinya bukanlah sebuah kursus. Sebuah powerpoint bukanlah kursus.
+
+
+<!----><a name="interactive-tutorials-vs-other-stuff"></a>
+##### Tutorial Interaktif vs. Hal-hal lain
+
+Jika Anda dapat mencetaknya dan mempertahankan esensinya, itu bukan Tutorial Interaktif.
+
+
+<!----><a name="automation"></a>
+### Otomatisasi
+
+- Pemformatan penegakan aturan otomatis melalui [GitHub Actions](https://github.com/features/actions) gunakan [fpb-lint](https://github.com/vhf/free-programming-books-lint) (lihat [`.github/workflows/fpb-lint.yml`](.github/workflows/fpb-lint.yml)).
```

Comment:
`.github/workflows/fpb-lint.yml` link doesn't work in any CONTRIBUTING files, for now you can fix it in this file

Reply:
@LuigiImVector you've found another bug :rocket: present in all CONTRIBUTING files since were moved to docs folder in #6614. How to adress it? The best way is always a relative URL so... 

```markdown
- Pemformatan penegakan aturan otomatis melalui [GitHub Actions](https://github.com/features/actions) gunakan [fpb-lint](https://github.com/vhf/free-programming-books-lint) (lihat [`.github/workflows/fpb-lint.yml`](../.github/workflows/fpb-lint.yml)).
```

Perhaps it is better deal with it in other PR, I thought, giving context. I can't believe it how I didn't detect it in #6625 :worried:

Reply:
You are right, the best way is relative URL. I am doing PR to fix this bug, I think you better fix the bug in this PR (only for Indonesian)